FhirToDataLake/deploy/templates/FhirSynapsePipelineTemplate.json

```diff
@@ -53,6 +53,34 @@
                 "description": "The name of the container to store job and data."
             }
         },
+        "filterScope":{
+            "type": "string",
+            "defaultValue": "System",
+            "metadata": {
+                "description": "The Scope you want to export data, System or Group."
+            }
+        },
+        "groupId":{
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The group id you want to export data for Group Scope."
+            }
+        },
+        "requiredTypes":{
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The resource type you want to export. Will continuous export all resource types if not specified."
+            }
+        },
+        "typeFilters":{
+            "type": "string",
+            "defaultValue": "",
+            "metadata": {
+                "description": "The type filters you want to filter the export data. Will continuous export all resource types if not specified."
```

Comment:
Seems `Will continuous export all resource types if not specified` is not suitable for typeFilter field?

Reply:
Refined.